src/app/content/highlights/components/cardStyles.ts

```diff
@@ -97,10 +97,11 @@ const fadeInAnimation = css`
 `;
 
 export const mainCardStyles = css`
-  ${(props: CardProps) => !props.isActive && (!props.data || !props.data.annotation)
-    ? 'display: none;'
-    : 'display: block;'}
+  ${(props: CardProps) => (!props.isActive && (!props.data || !props.data.annotation)) || props.isHidden
+    ? 'visibility: hidden;'
+    : 'visibility: visible;'}
```

Comment:
@TomWoodward Looks like `visibility: hidden` will continue to work like `display: none` and prevent if from being read by a screenreader: https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#accessibility_concerns